modules/standard/Math.chpl

```diff
@@ -1232,4 +1232,64 @@ module Math {
     extern proc yn(n: c_int, x: real(64)): real(64);
     return yn(n.safeCast(c_int), x);
   }
+
+  @chpldoc.nodoc
+  private inline proc fmaImplExternCall(x: real(?w), y: real(w), z: real(w)) {
+    extern proc fmaf(x: real(32), y: real(32), z: real(32)): real(32);
+    extern proc fma(x: real(64), y: real(64), z: real(64)): real(64);
+    var ret: x.type;
+    select x.type {
```

Comment:
I don't remember off-hand if we param fold `select` statements. I'd probably have just used conditionals, since those definitely are param folded.

Reply:
Confirming that yes, we do param-fold select statements.

Reply:
It's great that I haven't changed it yet, then!